### PR TITLE
KEYCLOAK-6943 Upgrade migration guide for clarify that switch 'Exclud…

### PR DESCRIPTION
**Pull request by mposolda**
…e Session State From Authentication Response' is available from 4.0.0.Beta1

### CODE REVIEW

File: upgrading/topics/keycloak/changes.adoc [hunk 1]
```diff
@@ -16,6 +16,10 @@ for the particular client in the {project_name} admin console, in client details
 described in <<_compatibility_with_older_adapters>>. There is the `Exclude Session State From Authentication Response` switch,
 which can be turned on to prevent adding the `session_state` parameter to the Authentication Response.
 
+NOTE: The parameter `session_state` was added in 3.4.2, however the switch `Exclude Session State From Authentication Response` was added
+in 4.0.0.Beta1. If your {project_name} server is on 3.4.2 or 3.4.3 and you have issues with `session_state` parameter, you will need
+to upgrade server to 4.0.0.Beta1 or newer.
```
Comment (matthewhelmke) [inline upgrading/topics/keycloak/changes.adoc]: ..."upgrade the server"...

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
Just one very small change request.

**Comment by mposolda** [comment]
@matthewhelmke Done, Thanks!

**Review by matthewhelmke — APPROVED** [review]

**Comment by matthewhelmke** [comment]
@mposolda Thank you!

**Pull request merged**